### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by HeyItsGwen**
Opening a pull request to check for stickler errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1126 @@
...
+  <meta charset='utf-8'>
+  <meta name='author' content='Gwen Hey'>
+  <meta name='viewport' content='width=device-width, initial-scale=1.0, shrink-to-fit=no'>
+  <meta name='description' content=''>
+  <meta name='keywords' content=''>
+
+  <title>Newsweek</title>
+
+  <!--Bootstrap CSS-->
+  <link rel='stylesheet' href='./css/bootstrap.min.css' type='text/css'>
```
Comment (suretrust) [inline index.html:14]: Using bootstrap's CDN is much better and faster. But not a deal-breaker.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1126 @@
...
+<body>
+
+  <nav class='container-fluid'>
+
+    <div class='navRow row'>
+      <div id='navDate' class='col-lg-4'>
+        Wed, Nov 6, 2019
+      </div>
+      <div id='navLogo' class=-lg-4'>
+        <div class='logoDiv'>'col-5 col
```
Comment (suretrust) [inline index.html]: There's some error with this line, kindly fix.

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi @HeyItsGwen,

#### Great job thus far! :raised_hands: Just a few suggestions to wrap this up:

- [x] Please see my comments on your code.

**Large screen**
- [x] The total height of the three images and captions by the right is equal to the height of the ones on the left. Kindly fix as appropriate, refer to the images below.

height
height2

- [x] The navbar should be sticky. (The sticky navbar is smaller) Please refer to the image below:

nav

**Medium screen**

- [x] The navbar has some alignment issues and should be sticky. Please see the image below:

nav2

**Small screen**
- [x] The navbar has some alignment issues and should be sticky. Please see the image below:

nav3

**Kindly submit for another code review once these issues have been fixed.** _Happy coding! :tada:_

**Comment by HeyItsGwen** [comment]
Thanks, Saheed! Fixed, validated, and resubmitting 😃

**Review by Redvanisation — APPROVED** [review]
# Project approved ✔️ 

### Hello @HeyItsGwen  🌟 

### _Nice work on this project, it seems that you have met the requirements so I am approving your project. 👍_ 

### You may merge to master and submit a project completion form. 🚀 

### Good luck! 🍀

**Comment by HeyItsGwen** [comment]
Thank you, Radouane!

**Pull request merged**